### PR TITLE
`depends` -> `recommends`

### PR DESCRIPTION
**Pull request by shedaniel**
also changed my name in contributors for no reason.

### CODE REVIEW

**Comment by Sollace** [comment]
No.

Use this.
```
  "depends": {
    "fabricloader": ">=0.6.1",
    "fabric-api-base": ">=0.1.0",
    "fabric-events-lifecycle-v0": ">=0.1.0",
    "fabric-resource-loader-v0": ">=0.1.0",
    "minecraft": ">=1.15-"
  }
```

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -16,15 +16,17 @@
     "issues": "https://github.com/Prospector/ModMenu/issues"
   },
   "depends": {
-    "fabric": "*",
     "fabricloader": ">=0.6.1",
     "minecraft": ">=1.15-"
   },
+  "recommends": {
```
Comment (modmuss50) [inline src/main/resources/fabric.mod.json]: No, it needs to depend on the specfic module it uses.

**Review by modmuss50 — CHANGES_REQUESTED** [review]

**Comment by shedaniel** [comment]
I don't think modmenu depends on `api-base` or the `event-lifecycle`, but I have resolved the issue. Please check it out.

**Comment by Sollace** [comment]
> 
> 
> I don't think modmenu depends on `api-base` or the `event-lifecycle`, but I have resolved the issue. Please check it out.

I included api-base because I wasn't sure if it was needed or not. (it seems like everything requires that).

`event-lifecycle` was just because I assumed it's needed to the `initialize` entrypoint to work.

**Comment by Sollace** [comment]
This is fixing #82 by the way. (you can close it if you merge this)

**Comment by Prospector** [comment]
Implemented in dev

**Pull request closed without merging**